docs/content/tutorial/0-set-up-your-development-environment.mdx

```diff
@@ -207,6 +207,25 @@ For other distributions, you can check out [PostgreSQL’s website for more guid
 
 You can download PostgreSQL on your macOS using [the installer on their website](https://www.postgresql.org/download/macosx/).
 
+</TabItem>
+<TabItem value="docker" label="Docker">
+
+If you have Docker installed, you can quickly spin up a PostgreSQL instance with the following command:
+
+```bash
+docker run --name postgres -e POSTGRES_USER=postgres -e POSTGRES_PASSWORD=postgres -e POSTGRES_DB=medusa-docker -p 5432:5432 -d postgres
+```
+
+`--name` creates a new container named `postgres`.
+
+`-e` creates environment variables POSTGRES_USER, POSTGRES_PASSWORD and POSTGRES_DB. These will be used to set up a database named `medusa-docker` with username and password being `postgres`.
+
+`-p` maps the container port 5432 to the external host 5432. This allows you to connect to the database server from the outside.
+
+`-d` enables Docker to run the container in the background.
+
+The last seciton of the command, `postgres` grabs the latest postgres image from the Docker hub.
```

Comment:
Please check the text in the suggestion to make sure it's all correct relative to Docker since I don't have much experience with it.

```suggestion
Where:

- `--name` creates a new container named `postgres`.
- `-e` creates environment variables `POSTGRES_USER`, `POSTGRES_PASSWORD` and `POSTGRES_DB`. These will be used to set up a database named `medusa-docker` with the username and password being `postgres`.
- `-p` maps the container port `5432` to the external host `5432`. This allows you to connect to the database server from outside of the container.
- `-d` enables Docker to run the container in the background.
- The last section of the command, `postgres` grabs the latest postgres image from the Docker hub.
```

Reply:
I've used Postgre with Docker a lot lately so I'm very sure that they are correct.

Reply:
I meant to check the changes I made and not the original text that you wrote 😄